### ISSUE TITLE
Stuck at "fb0: Switching to inteldrmfb from EFI VGA" - Ubuntu Installation

### ISSUE THREAD

**Issue by Jiya-Ull-Haq** [t2linux/wiki#111]
- Installed T2Linux Ubuntu Iso and made a bootable USB drive using BalenaEtcher.
- Plugged it into my `MacBook Air 2019 13"`
- selected `Try Ubuntu FS without Installing`. 
- Then after a while, it stuck at `fb0: Switching to inteldrmfb from EFI VGA`

**Comment by AdityaGarg8** [comment]
May I know the ISO you are using?

**Comment by Jiya-Ull-Haq-Shaik** [comment]
ubuntu-20.04-5.7.19-mbp.iso

**Comment by Jiya-Ull-Haq-Shaik** [comment]
Also tried `ubuntu-20.04-5.7.19-mbp-alt.iso` Same issue.

More info:
There is no OS on that Macbook, 128GB Unformatted Storage.
Trying to convert it to Linux Machine.

**Comment by Jiya-Ull-Haq-Shaik** [comment]
If I have to Format the Disk.

I'm having the following:
- External SSD with Bootable Windows OS,
- Other Macbook Air 2019 which is also T2.

**Comment by AdityaGarg8** [comment]
Is Install FS working for ubuntu-20.04-5.7.19-mbp.iso?

**Comment by AdityaGarg8** [comment]
If not, please share the output.

**Comment by Jiya-Ull-Haq-Shaik** [comment]
> Is Install FS working for ubuntu-20.04-5.7.19-mbp.iso?

No.
Same error in both ubuntu-20.04-5.7.19-mbp.iso and ubuntu-20.04-5.7.19-mbp-alt.iso

**Comment by Jiya-Ull-Haq-Shaik** [comment]
> If not, please share the output.

![IMG_C01D5AC168A6-1](https://user-images.githubusercontent.com/60106386/128592861-613b86e6-b5e1-4341-bacb-aeff80705556.jpeg)

**Comment by AdityaGarg8** [comment]
Let me see if something can be done. @Redecorating, could you help him?

**Comment by Redecorating** [comment]
I don't know why it's doing that, but here are a few things you can try:

Check if the caps lock light works when you press the caps lock button, then try plugging in an external monitor, if the external monitor doesn't work then try booting with the monitor plugged in and the macbook lid closed. 

You can also try the `nomodeset` kernel argument, by pressing `e` on "Try Ubuntu FS without Installing" and then use arrow keys to move to the line that starts with `linux`, add `nomodeset` to the end of that line, and finally press f10.

**Comment by Jiya-Ull-Haq-Shaik** [comment]
1. Plugged an external VGA monitor.
Result: `fb0: Switching to inteldrmfb from EFI VGA` (just like the picture above).

2. Did it again then closed the lid
Result: `fb0: Switching to inteldrmfb from EFI VGA` (just like the picture above).

3. Did the process again, Pressing CapsLock doesn't turn on the CapsLock light. 
Result was still the same.

4. Added `nomodeset` at the end of the Linux line and pressed f10
- VGA was still plugged in,
- Pressing CapsLock doesn't turn on the CapsLock light. 
- Result was different this time.
![IMG_FD2D54DCB964-1](https://user-images.githubusercontent.com/60106386/128594604-a1fc0e7a-5afd-4d6a-9af1-7908c31e6344.jpeg)

**Comment by nikodunk** [comment]
So this is long shot but I ran into one of these issues too during bootup no matter if I used Fedora, Ubuntu and Manjaro (though slightly different specifics). Turned out using a different USB-C to USB-A dongle solved it for me for all 3 images, everything else unchanged. I have _no_ idea why, but I reproduced the error before on one dongle and confirmed it was fixed with the other USBC to A dongle. Maybe give it a try. Also used Balena Etcher for everything.
